rest-helix/src/main/java/com/github/twitch4j/helix/TwitchHelix.java

```diff
@@ -728,6 +729,114 @@ HystrixCommand<UpdatedDropEntitlementsList> updateDropsEntitlements(
         UpdateDropEntitlementInput input
     );
 
+    /**
+     * Creates a new EventSub conduit.
+     *
+     * @param authToken  App access token.
+     * @param shardCount The number of shards to create for this conduit.
+     * @return ConduitList
+     */
+    @RequestLine("POST /eventsub/conduits")
+    @Headers({
+        "Authorization: Bearer {token}",
+        "Content-Type: application/json"
+    })
+    @Body("%7B\"shard_count\":{shard_count}%7D")
+    HystrixCommand<ConduitList> createConduit(
+        @Param("token") String authToken,
+        @Param("shard_count") int shardCount
+    );
+
+    /**
+     * Gets the conduits for a client ID.
+     *
+     * @param authToken App access token.
+     * @return ConduitList
+     */
+    @RequestLine("GET /eventsub/conduits")
+    @Headers("Authorization: Bearer {token}")
+    HystrixCommand<ConduitList> getConduits(
+        @Param("token") String authToken
+    );
+
+    /**
+     * Updates a conduit’s shard count.
+     * <p>
+     * To delete shards, update the count to a lower number, and the shards above the count will be deleted.
+     * For example, if the existing shard count is 100, by resetting shard count to 50, shards 50-99 are disabled.
+     *
+     * @param authToken  App access token.
+     * @param conduitId  The ID of the conduit to update.
+     * @param shardCount The updated shard count for the conduit.
+     * @return ConduitList
+     */
+    @RequestLine("PATCH /eventsub/conduits")
+    @Headers({
+        "Authorization: Bearer {token}",
+        "Content-Type: application/json"
+    })
+    @Body("%7B\"id\":\"{id}\",\"shard_count\":{shard_count}%7D")
+    HystrixCommand<ConduitList> updateConduit(
+        @Param("token") String authToken,
+        @Param("id") String conduitId,
+        @Param("shard_count") int shardCount
```

Comment:
```suggestion
        @NotNull @Param("id") String conduitId,
        @NotNull @Param("shard_count") int shardCount
```

Reply:
edited this suggestion bc of https://github.com/twitch4j/twitch4j/pull/923#discussion_r1548590223